public/stylesheets/app.css

```diff
@@ -19,10 +19,26 @@ ui.container {
 }
 
 .jokeTypeButton{
```

Comment:
format CSS properly

Reply:
Can you pls explain what do you mean by "format CSS"